src/components/Charts/zoneUtils.js

```diff
@@ -37,10 +38,17 @@ const ZONES_POSITIVE_RATE = getHighchartZones(POSITIVE_TESTS);
 const ZONES_HOSPITAL_USAGE = getHighchartZones(HOSPITAL_USAGE);
 
 export const optionsRt = (data, endDate) => {
-  const { x, y } = lastValidPoint(data);
   const zones = ZONES_RT;
   const [minY, maxY] = [0, getMaxY(data)];
   const [minYAxis, maxYAxis] = getYAxisLimits(minY, maxY, zones);
+
+  // Split the Rt data in two sets, recent data and previous data so we can
+  // dot the line for most recent dates. They overlap to minimize the gap
+  // between the series.
+  const dateRecent = moment().subtract(7, 'days').toDate();
```

Comment:
Can you merge / rebase develop and use my `RT_TRUNCATION_DAYS` constant from https://github.com/covid-projections/covid-projections/commit/342027752ed47d5070ad73a190af4566a39543dd ?